Doc/c-api/init_config.rst

```diff
@@ -1602,6 +1602,53 @@ customized Python always running in isolated mode using
 :c:func:`Py_RunMain`.
 
 
+Get the current Python configuration
+====================================
+
+Get a configuration option where *name* is the name of a :c:type:`PyConfig`
+member.
+
+Some options are read from the :mod:`sys` attributes. For example, the option
+``"argv"`` is read from :data:`sys.argv`.
+
+
+.. c:function:: int PyConfig_Get(const char *name, PyObject **value)
```

Comment:
I assume this has to be run with the GIL held and after initialization?

Reply:
Right, I completed the doc.